docs/adapting-settings.md

```diff
@@ -48,6 +48,13 @@ A secret key used as salt for cryptographic parts.
 
 The site identifier. It is used to attached some database configuration, metrics...
 
+### SITE_TERMS
+
+**default**: `'default'`
+
+The site terms in markdown. It can be either an URL to a markdown content or a local path.
```

Comment:
s/It can be either an URL to a markdown content or a local path./It can be either an URL or a local path to a markdown content./

Reply:
s/terms/terms and conditions/